MOVIShield.cpp

```diff
@@ -45,11 +45,13 @@
 
 MOVI::MOVI()
 {
+    usehardwareserial=false;
     construct(ARDUINO_RX_PIN, ARDUINO_TX_PIN, false);
 }
 
-MOVI::MOVI(bool debugonoff)
+MOVI::MOVI(uint8_t debugonoff)
```

Comment:
Sure, that way is totally fine - those of us needing HardwareSerial just have to pass one additional parameter that may well always be hardcoded to false, but that's easy enough to live with.

Would you like me to submit an amended pull request?